tests/unit/wordpress_mock.py

```diff
@@ -645,6 +645,13 @@ def _mock_chown_uploads(self, _cmd):
         """Simulate ``chown`` command execution in the container."""
         return ExecProcessMock(return_code=0, stdout="", stderr="")
 
+    @_exec_handler.register(lambda cmd: cmd[:3] == ["wp", "core", "update-db"])
+    def _mock_wp_update_database(self, _cmd):
+        """Simulate ``wp core update-db`` command execution in the container."""
+        if self._fail_wp_update_database:
```

Comment:
What happens if `_fail_wp_update_database_` is not initialised? You should probably initialise it in the constructor.

Reply:
Done! Thanks!!